### DISCUSSION TITLE
Multi-GPU on WSL2 works

### DISCUSSION THREAD

**Discussion by rosslwheeler** [karpathy/llm.c#271] [General]
Just tested it and it works well.

**Comment by colinbrace** [comment]
Nice! Though I have to ask -- what hardware/software configuration do you have that uses WSL2 _and_ has multiple cuda-enabled GPU's?

**Comment by rosslwheeler** [comment]
Hah! Yes, it's not a laptop - Dell 7865 with 2 A5500's.
